source/add-jupyter/customize-attributes.rst

```diff
@@ -0,0 +1,129 @@
+.. _add-jupyter-customize-attributes:
+
+Customize Attributes
+====================
+
+Now we will customize the app to work on a given cluster. Be sure that you
+walked through :ref:`add-jupyter-software-requirements` for the given cluster
+ahead of time.
+
+Modify Form File
+----------------
+
+The main responsibility of the ``form.yml`` file is for defining the attributes
+(their values or HTML form elements) used when generating the batch script.
+
+#. We will begin by adding a cluster for the Jupyter app to use. You do this by
+   editing the ``form.yml`` in your favorite editor as such:
+
+   .. code-block:: yaml
+
+      # form.yml
+      ---
+
+      ...
+
+      cluster: "my_cluster"
+
+      ...
+
+   where we replace ``my_cluster`` with a valid cluster that corresponds to a
+   cluster configuration file located under
+   :file:`/etc/ood/config/clusters.d/{cluster}.yml`.
```

Comment:
The brackets here don't render into sphinx html. Maybe they need to be escaped? As a result, it's unclear that you're directing people to use the name of the yml file. Maybe change the name of the file to `my_cluster.yml` to indicate that the aforementioned `my_cluster` variable will be replaced with this name.

Reply:
The brackets cause the word `cluster` to be emphasized indicating that it is a variable and should be replaced.

For more information, please see:

http://www.sphinx-doc.org/en/stable/markup/inline.html#role-file

Reply:
I see it now. It's not very noticeable in practice though, and it's confusing because you're trying to tell people that "my_cluster" should be replaced with "{cluster}.yml". I think the terms should be the same at least.

![cluster](https://user-images.githubusercontent.com/2374718/30484382-36ca2f72-99f8-11e7-9de3-39f70ebb18e2.png)